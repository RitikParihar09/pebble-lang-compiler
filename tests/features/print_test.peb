// Test: print vs display functionality
// This test demonstrates the difference between print() and display()

// Integer tests
var num = 42
print("Integer with print(): ")
print(num)
display("Integer with display(): ")
display(num)

// String tests
var str = "Hello, World!"
print("String with print(): ")
print(str)
display("String with display(): ")
display(str)

// Float tests
var pi = 3.14159
print("Float with print(): ")
print(pi)
display("Float with display(): ")
display(pi)

// Character tests
var letter = 'A'
print("Character with print(): ")
print(letter)
display("Character with display(): ")
display(letter)

// Multiple prints on one line
print("Multiple ")
print("prints ")
print("on ")
print("one ")
print("line")
display("")  // Just to add a newline at the end
