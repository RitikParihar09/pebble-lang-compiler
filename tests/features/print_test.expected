Integer with print(): 42
Integer with display(): 
42
String with print(): Hello, World!
String with display(): 
Hello, World!
Float with print(): 3.14
Float with display(): 
3.14
Character with print(): A
Character with display(): 
A
Multiple prints on one line
